### PR TITLE
Syndicate AI Announcements!

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
The syndicate, using their extreme tech and skills, has managed to infiltrate the comms system! Now the syndicate can yell at the crew in real time

## Why It's Good For The Game
Fun variety for the announcer since theres only like 3 announcers currently, also i really like the syndicate voicelines

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

syndiemessup

https://user-images.githubusercontent.com/73374039/182741915-b19f3f5c-b8e8-4031-9ef0-842a0e26cc84.mp4

(demonstrating one of the station welcome and centcom alert tracks)

(i cant get station traits to work locally, testing was done by directly setting the announcer but im sure it will work fine)

</details>

## Changelog
:cl:
:add: syndicate announcer trait
soundadd: syndicate announcer sfx
/:cl:

### CODE REVIEW

**Comment by Miliviu** [comment]
Please, add at least one audio to testing photograh and procedure!

**Comment by etherware-novice** [comment]
@Miliviu audio demo inserted!

File: sound/ai/syndicate/script.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+commandreport/1.ogg you got mail
+commandreport/2.ogg nano tray sen command has something to say listen up nerds
+commandreport/3.ogg intercepted message from cent com
+commandreport/4.ogg nano tray sen has an important message
+commandreport/5.ogg new message in communication console
+
+welcome/1.ogg welcome to sin dee kite station
+welcome/2.ogg nano tray sen can not save you now
+welcome/3.ogg all your station are belong to us
```
Comment (itsmeow) [inline sound/ai/syndicate/script.txt]: Too meme for RP server

File: sound/ai/syndicate/script.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+intercept.ogg 		nano tray sen command has detected our agents aboard the station
+ionstorm.ogg 		artificial core minor subversion successful
+meteors.ogg 		watch for flying rocks
+outbreak5.ogg 		all nuclear ops your mission is cancelled return to base
+outbreak7.ogg 		viral culture has been implanted on ship
+poweroff.ogg 		implanted corrupt code in lighting subroutine
+poweron.ogg 		nano tray sen command is smarter than we thought
+radiation.ogg 		radioactive frequencies have been broadcasted to the station
+shuttlecalled.ogg 	our work here is done
+shuttledock.ogg 	your precious emergency shuttle is here get on
```
Comment (itsmeow) [inline sound/ai/syndicate/script.txt:14]: this one is weird idk

File: sound/ai/syndicate/script.txt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+shuttlerecalled.ogg 	ready for round two
```
Comment (itsmeow) [inline sound/ai/syndicate/script.txt:15]: same for this

File: sound/ai/syndicate/script.txt (hunk 2 above)
Comment (etherware-novice) [reply sound/ai/syndicate/script.txt:14]: which one

File: sound/ai/syndicate/script.txt (hunk 3 above)
Comment (etherware-novice) [reply sound/ai/syndicate/script.txt:15]: so uh the mentality for these are like
shuttle called is like "we've managed to damage the station to a irriparable state mission complete"
docked is mocking the crew about their shuttle and how "pwecious it is"
recalled is like "alright we're ready to throw more shtuff at the crew until you break again"

File: sound/ai/syndicate/script.txt (hunk 2 above)
Comment (itsmeow) [reply sound/ai/syndicate/script.txt:14]: your precious emergency shuttle is here get on

File: code/datums/announcers/syndie_announcer.dm [hunk 4]
```diff
@@ -0,0 +1,41 @@
+/datum/centcom_announcer/syndie
+	welcome_sounds = list('sound/ai/syndicate/welcome/1.ogg',
+							'sound/ai/syndicate/welcome/2.ogg',
+							'sound/ai/syndicate/welcome/3.ogg',
```
Comment (itsmeow) [inline code/datums/announcers/syndie_announcer.dm:4]: you didn't actually remove welcome3. Also rename the higher ones to fill the gap

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Licence for these sounds is:
image
We can't use this probably

**Comment by etherware-novice** [comment]
its already in game tho, i deliberately used the same ai as in sound/ambience/antag/nukeops.ogg

**Comment by PowerfulBacon** [comment]
The sounds in ops.ogg sounds different to the voice lines provided.
Existing potential breaches in copyright doesn't allow for additional ones to be implemented.

**Comment by etherware-novice** [comment]
the sound is different because theres filters i dont know how to add!! its the same generator i swear cmon its abandonware it already exists in the code theres no way theres any copyright issues please

**Comment by PowerfulBacon** [comment]
Closing due to potential copyright issues, its unlikely to cause an issue but its not 100% confirmed that we can use this.

**Pull request closed without merging**